devicetypes/tonesto7/nest-thermostat.src/nest-thermostat.groovy

```diff
@@ -340,7 +344,7 @@ def deviceVerEvent() {
 	if (devVer()) {
     	def cur = parent?.latestTstatVer().ver.toString()
     	def ver = (cur != devVer().toString()) ? "Device Type:\nv${devVer()}(Latest: v${cur})" : "Device Type:\nv${devVer()}(Current)"
-    	sendEvent(name: 'devTypeVer', value: ver, displayed: false, isStateChange: true)
```

Comment:
Leaving it off has ST decide if it changed or not.

If you put it in, you are deciding.

I wanted to let ST decide it has or has not changed

On Mar 23, 2016, at 9:04 AM, Tonesto7 <notifications@github.com<mailto:notifications@github.com>> wrote:


In devicetypes/tonesto7/nest-thermostat.src/nest-thermostat.groovy<https://github.com/tonesto7/nest-manager/pull/34#discussion_r57154929>:

> @@ -340,7 +344,7 @@ def deviceVerEvent() {
>       if (devVer()) {
>       def cur = parent?.latestTstatVer().ver.toString()
>       def ver = (cur != devVer().toString()) ? "Device Type:\nv${devVer()}(Latest: v${cur})" : "Device Type:\nv${devVer()}(Current)"
> -     sendEvent(name: 'devTypeVer', value: ver, displayed: false, isStateChange: true)


What would be the reason for using isStateChange vs. not using it?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/tonesto7/nest-manager/pull/34/files/71738294e6f15465e5de7ddacf369c0b6f73c063#r57154929>